src/funtest/features/download.feature

```diff
@@ -26,6 +26,17 @@ Feature: Download a Candidate Version
     Then a redirect to "http://dl.bintray.com/groovy/maven/apache-groovy-binary-2.4.7.zip" is returned
     And an audit entry for groovy 2.4.7 UNIVERSAL is recorded for LinuxX64
 
+  Scenario: Retrieve checksum when not found
```

Comment:
I think checksum might be a feature, so should probably live in a feature file of it's own.